components/search/filter/CategoryButtonContainer.stories.tsx

```diff
@@ -0,0 +1,52 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import { SearchParamsType } from '@/types/searchPageTypes';
+import CategoryButtonContainer from './CategoryButtonContainer.';
+
+const meta: Meta<typeof CategoryButtonContainer> = {
+  title: 'Search/Filter/CategoryButtons',
+  component: CategoryButtonContainer,
+  decorators: [
+    (Story) => (
+      <div style={{ maxWidth: '600px' }}>
+        <Story />
+      </div>
+    ),
+  ],
+};
+
+export default meta;
+
+type Story = StoryObj<typeof meta>;
+
+const params: SearchParamsType = {
+  query: '',
+  category: '',
+  subcategory: '',
+  limit: 10,
+  cursor: '',
+};
+
+export const CategoryButtonsDefault: Story = {
+  args: {
+    searchParams: params,
+  },
+};
```

Comment:
I'm really annoyed that the text does not update on this story but I know why we're moving ahead with it. Can you just pass the `setSearchParams` arg as an `fn()` this will make it log the action in the story when people try to type

![Screenshot 2024-07-31 at 16 07 06](https://github.com/user-attachments/assets/a8ff201b-55f0-4f2b-b070-e65c45f330ce)